apis/src/main/kotlin/org/yapp/apis/auth/strategy/withdraw/GoogleWithdrawStrategy.kt

```
@@ -0,0 +1,32 @@
+    private val log = KotlinLogging.logger {}
+
+    override fun getProviderType(): ProviderType = ProviderType.GOOGLE
+
+    override fun withdraw(request: WithdrawStrategyRequest) {
+        val googleRefreshToken = request.googleRefreshToken
+            ?: throw AuthException(AuthErrorCode.GOOGLE_REFRESH_TOKEN_NOT_FOUND, "Google Refresh Token이 존재하지 않습니다.")
+
+        try {
+            googleApiManager.revokeToken(googleRefreshToken as String)
+            log.info { "Google refresh token revoked successfully for user ${request.userId}" }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)